public/vendor/formatjs/integrations.js

```diff
@@ -6,3 +6,9 @@ window.HandlebarsIntl = HandlebarsIntl;
 
 import ReactIntl from 'react-intl';
 window.ReactIntl = ReactIntl;
+
+import IntlRelativeFormat from 'intl-relativeformat';
+window.IntlRelativeFormat = IntlRelativeFormat;
+
+import IntlMessageFormat from 'intl-messageformat';
+window.IntlMessageFormat = IntlMessageFormat;
```

Comment:
So this is going to be problematic. Why do these need to be exposed? Can we add an `__addLocaleData()` function onto Ember Intl in some way? See #165 for the problem we currently have on the site.

The only reason that we have `intl-messageformat` and `intl-relativeformat` in the site's package.json is to optimize the build such that we can combine all the high level integrations into a single ES6 bundled module. Ember Intl supports being imported as an ES6 module like the rest of the integration libs, right?